code/modules/ruins/spaceruin_code/DJstation.toml

```diff
@@ -0,0 +1,13 @@
+directory = "_maps/RandomRuins/SpaceRuins/DJstation/"
+
+[rooms.radioroom]
```

Comment:
where do you define the direction they are and the offset?

Reply:
There’s an object on the map, /obj/modular_map_connector. It uses that to calculate the offset using the same method as shuttles do with the docking port. No support for rotating the modules, you build them in the orientation you want them.

Reply:
shouldn't that also go in these confi gfiles instead? now it's worse because it's split over two things

Reply:
Because if it was numbers in the config you would have to manually calculate the offsets for each module, we used to do this with shuttles and the automatic calculation is much less hassle